rust/operator-binary/src/crd.rs

```diff
@@ -143,14 +143,26 @@ impl AutoTlsCa {
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "camelCase")]
-pub struct AdditionalTrustRoot {
+pub enum AdditionalTrustRoot {
+    /// Reference (name and namespace) to a Kubernetes ConfigMap object where additional
+    /// certificates are stored.
+    /// The extensions of the keys denote its contents: A key suffixed with `.crt` contains a stack
+    /// of base64 encoded DER certificates, a key suffixed with `.der` contains a binary DER
+    /// certificate.
+    ConfigMap {
+        #[serde(flatten)]
+        config_map: ConfigMapReference,
+    },
```

Comment:
Nit: From what I remember, we could use tuple variants here:

```suggestion
    ConfigMap(ConfigMapReference),
```

But no need to try that right now, it shouldn't affect the API either way.

Reply:
done and successfully tested (a94f08b8f86b46cc137e8266fdfd009ef89d7334)